decentraland/screens/ListLandForSaleScreen.js

```diff
@@ -147,29 +156,34 @@ export class ListLandForSaleScreen extends React.Component {
   };
 
   render() {
-    const { landForSaleList } = this.props;
+    const { assetsForSale } = this.props;
+    const { list, loadingInProgress } = assetsForSale;
+
     return (
       <LandForSaleList
-        landForSaleList={landForSaleList}
+        landForSaleList={list}
         renderItem={this._renderItem}
+        loadingInProgress={loadingInProgress}
       />
     );
   }
 }
 
 ListLandForSaleScreen.propTypes = {
-  landForSaleList: PropTypes.array.isRequired,
+  assetsForSale: PropTypes.object.isRequired,
```

Comment:
I wonder if we should have kept what's now called `assetsForSale` as an array and had a separate prop not inside the object for the loading boolean

Reply:
Hmmm,
Both `assetsForSale` and `accountInfo` has their loading flags as an inside prop. I don't have a strong opinion if we should keep that way or extract the flags to the same level.
I think that keep that way has a pro to keep cohesion but it adds more complexity since it adds more nested objects.

Reply:
Okay - that makes sense and definitely won't gate merging this PR. This will be an interesting topic to ponder together going forward. I think having them as separate loading states might force us to consider whether we have a reasonable amount of loading states. We might even move all loading states at some point into a dedicated loading state section of state.

Reply:
Should we extract this to an issue or just keep it in mind?

Reply:
I think that an issue will be better: https://github.com/tasitlabs/tasit/issues/197

Reply:
Okay, sounds good. Resolving this.